core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -566,6 +567,7 @@ object KafkaConfig {
   val InterBrokerProtocolVersionProp = "inter.broker.protocol.version"
   val InterBrokerListenerNameProp = "inter.broker.listener.name"
   val ReplicaSelectorClassProp = "replica.selector.class"
+  val LiNumMinOriginalAliveReplicasProp = "li.num.min.original.alive.replicas"
```

Comment:
nit: the "Num"/"num" part is unnecessary (e.g., "min.insync.replicas") and can be dropped, but if present, it should come after the "min" (i.e., "minimum number" not "number minimum").

Reply:
fixed